nms/app/packages/magmalte/app/views/subscriber/SubscriberAddDialog.js

```diff
@@ -157,19 +167,21 @@ function parseSubscriberFile(fileObj: File) {
           .map(item => item.trim())
           .filter(Boolean)) {
           const items = line.split(',').map(item => item.trim());
-          if (items.length != 7) {
+          if (items.length > SUB_MAX_FIELDS) {
             reject('failed parsing ' + line);
             return;
           }
           subscribers.push({
-            name: items[0],
-            imsi: items[1],
-            authKey: items[2],
-            authOpc: items[3],
-            state: items[4] === 'ACTIVE' ? 'ACTIVE' : 'INACTIVE',
-            dataPlan: items[5],
-            apns: items[6]
-              .split('|')
+            name: items[SUB_NAME_OFFSET],
```

Comment:
I think it would be cleaner to have a function to parse an individual subscriber and then just call it on all the lines here. You could then pass in the format to that function and have this all be more resilient to change.